### PR TITLE
Avivash/different font weights

### PR DESCRIPTION
**Pull request by avivash**
# Description

Adding `font-weight`s  to the fonts to fix rendering issues in iOS

## Link to issue

https://discord.com/channels/478735028319158273/1039954639585546270/1039962081346191432

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue)

## Screenshots/Screencaps

image

### CODE REVIEW

File: src/global.css [hunk 1]
```diff
@@ -6,23 +6,23 @@
   font-family: 'UncutSans_Regular';
```
Comment (icidasset) [inline src/global.css:6]: I would make this `UncutSans` for all of the `@font-face`s, that way you don't need the change the font-family all the time.

Comment (avivash) [reply src/global.css:6]: Sorry, I'm not sure if I follow. If I make them all the same `font-face` the fonts don't render correctly 🤔 
image

Comment (icidasset) [reply src/global.css:6]: Hmm, that's odd, that should work. I'll give it a go in a few hours 👀

Comment (icidasset) [reply src/global.css]: I made some changes, seems to be ok now 👍

Comment (icidasset) [reply src/global.css]: Sorry for the formatting commits, forgot to disable that 😅 
Let me know if I need to squash those commits and if it works for you.

Comment (avivash) [reply src/global.css]: Thanks! That's an interesting fix 🤔 shouldn't it just default to the last @font-face defined in the list if they all have the same name? https://github.com/webnative-examples/webnative-app-template/pull/87/files#diff-6a7ee04012624d3db7b781ffd397b00ae6dff2154c43fc5f20624ebc0fdb4b5aR22

Comment (avivash) [reply src/global.css]: I've never used multiple font-faces that all have the same name, but point to different files. I'll try to find some documentation on this 👀

**Pull request merged**